src/content/content.js

```
@@ -65,71 +84,122 @@ injectJs(chrome.runtime.getURL("injected.js"));
+        detail: request.code,
+      });
+      window.dispatchEvent(loadTestEvent);
+      break;
+
+    case "filename":
+      sendResponse({ filename: filename });
+      logDebug("Filename request handled", filename);
+      break;
+
+    case "unloadHeader":
+      const headerCode = document.querySelector("#header_content")?.value || "";
```

Biome (1.9.4)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Unsafe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)